eventmesh-sdks/eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumer.java

```diff
@@ -128,8 +129,9 @@ public void subscribe(final List<SubscriptionItem> subscriptionItems) {
 
         addSubscription(subscriptionItems, SDK_STREAM_URL);
 
-        CloudEvent subscription = EventMeshCloudEventBuilder.buildEventSubscription(clientConfig, EventMeshProtocolType.EVENT_MESH_MESSAGE, null,
-            subscriptionItems);
+        CloudEvent subscription =
+            EventMeshCloudEventBuilder.buildEventSubscription(clientConfig, EventMeshMessageProtocolType.EVENT_MESH_MESSAGE, null,
+                subscriptionItems);
         synchronized (this) {
```

Comment:
![image](https://github.com/apache/eventmesh/assets/41445332/712ff47d-bf9c-49aa-9c53-98a455965452)

![image](https://github.com/apache/eventmesh/assets/41445332/496d6d0a-2f6e-431c-9ed4-bdaf4c2d64b7)

The former one seems confusing and the later one will be enough.

Reply:
done